deployment.txt

```diff
@@ -0,0 +1,394 @@
+
+Deploying AuraToken contract with hash 0xf14633cca8a0bad1421683f25e751b3e6fa25a473c460653b001e71ee83e95a7 from 0x30019eB135532bDdF2Da17659101cc000C73c8e4 with gas price 0 Gwei
```

Comment:
I would suggest to remove it or change the name to something like 
goerli-deployment.log